app/0.14.x/getting-started/configuring-a-service.md

```diff
@@ -135,3 +140,5 @@ Go to [Enabling Plugins &rsaquo;][enabling-plugins]
 [enabling-plugins]: /{{page.kong_version}}/getting-started/enabling-plugins
 [proxy-port]: /{{page.kong_version}}/configuration/#nginx-section
 [mockbin]: https://mockbin.com/
+
+[Back to TOC](#table-of-contents)
```

Comment:
The trailing "Back to TOC" looks strange on the Getting Started pages (at least on all of them except the last one).

![configuring_a_service_-_v0_14_x___kong_-_open-source_api_management_and_microservice_management](https://user-images.githubusercontent.com/94904/47589067-fe168300-d91c-11e8-867c-42be74e3daf6.jpg)